framework/web/Session.php

```diff
@@ -339,40 +345,40 @@ public function regenerateID($deleteOldSession = false)
 
     /**
      * Gets the name of the current session.
-     * This is a wrapper for [PHP session_name()](https://secure.php.net/manual/en/function.session-name.php).
+     * This is a wrapper for [PHP session_name()](https://www.php.net/manual/en/function.session-name.php).
      * @return string the current session name
      */
     public function getName()
     {
-        return session_name();
+        return (string) session_name();
     }
 
     /**
      * Sets the name for the current session.
-     * This is a wrapper for [PHP session_name()](https://secure.php.net/manual/en/function.session-name.php).
+     * This is a wrapper for [PHP session_name()](https://www.php.net/manual/en/function.session-name.php).
      * @param string $value the session name for the current session, must be an alphanumeric string.
      * It defaults to "PHPSESSID".
      */
     public function setName($value)
     {
         $this->freeze();
-        session_name($value);
+        session_name((string) $value);
         $this->unfreeze();
     }
 
     /**
      * Gets the current session save path.
-     * This is a wrapper for [PHP session_save_path()](https://secure.php.net/manual/en/function.session-save-path.php).
+     * This is a wrapper for [PHP session_save_path()](https://www.php.net/manual/en/function.session-save-path.php).
      * @return string the current session save path, defaults to '/tmp'.
      */
     public function getSavePath()
     {
-        return session_save_path();
+        return (string) session_save_path();
```

Comment:
When it returns `false` we'll get empty string so the session may be written into current directory.

Reply:
@samdark it's BC because not declared

Reply:
It changes behavior.

Reply:
@samdark we update returned types or normalize returned values. It's BC in any case.